### ISSUE TITLE
A Syntax question

### ISSUE THREAD

**Issue by jinyu121** [jwyang/faster-rcnn.pytorch#191]
https://github.com/jwyang/faster-rcnn.pytorch/blob/master/trainval_net.py#L265-L269

I know you mean, the learning rate in the dict should use `lr`. But in pure python, it is value-copy. For example: 

```python
a = 10
d = {"a": a * 10}
print(d)
a = 10000
print(d)
```

It will produce 

```
{'a': 100}
{'a': 100}
```

But in your code... well, it will _produce_ 

```
{'a': 100}
{'a': 100000}
```

I wonder why.

**Comment by jinyu121** [comment]
OK... Well, something called `adjust_learning_rate`. Close this issue.
